packages/core/src/L3/nodejs/tools/TypescriptSupport.ts

```diff
@@ -423,9 +424,9 @@ export class TypescriptSupport extends Construct {
       new ManifestEntry(this, "Scripts", {
         scripts: {
           compile: "tsc -p ./tsconfig.json",
-          clean: 'find . -name "*.js" -not -path "./node_modules/*" -delete && find . -name "*.d.ts" -not -path "./node_modules/*" -delete',
         },
       });
+      new CleanScriptPath(this, "CleanTsc", project.buildPath);
```

Comment:
```suggestion
      new CleanScriptPath(this, "CleanBuild", project.buildPath);
```

I think this is more descriptive, and the `id` values only need to be unique within the scope (`this`).